client/payment-details/summary/index.tsx

```diff
@@ -223,19 +223,40 @@ const PaymentDetailsSummary: React.FC< PaymentDetailsSummaryProps > = ( {
 			<CardBody>
 				<div className="payment-details-summary">
 					<div className="payment-details-summary__section">
-						<p className="payment-details-summary__amount">
+						<p
+							className="payment-details-summary__amount"
+							aria-label={ __(
+								'Payment amount',
+								'woocommerce-payments'
+							) }
+						>
 							<Loadable
 								isLoading={ isLoading }
 								placeholder="Amount placeholder"
 							>
-								{ formatCurrency(
-									charge.amount,
-									charge.currency,
-									balance.currency
+								{ renderStorePrice ? (
```

Comment:
Does this boolean mean "display headline number in merchant currency" or "show merchant currency label because shopper currency is different"?

Reply:
Answer: "show merchant currency label because shopper currency is different".

I've attempted to clarify this by naming the variable `isStoreAndShopperCurrencyMismatched` in fd93486.